addons/event_sms/tests/test_sms_schedule.py

```diff
@@ -17,14 +17,14 @@ def setUpClass(cls):
         cls.sms_template_sub = cls.env['sms.template'].create({
             'name': 'Test subscription',
             'model_id': cls.env.ref('event.model_event_registration').id,
-            'body': '${object.event_id.organizer_id.name} registration confirmation.',
-            'lang': '${object.partner_id.lang}'
+            'body': '{{ object.event_id.organizer_id.name }} registration confirmation.',
```

Comment:
Since the comment of jke (https://github.com/odoo/odoo/pull/70045#issuecomment-865784534), I have chosen arbitrarily one of both style.
Now that it is done is it really necessary to change this in all the templates ?
Seems unnecessary work for me.